typebinder_web_playground/src/lib.rs

```diff
@@ -83,47 +83,61 @@ impl<'a> Exporter for StringOutputter<'a> {
     }
 }
 
-pub struct AppState {
+pub struct Model {
     source: String,
     output: String,
 }
 
-pub enum AppMsg {
+pub enum Msg {
     ChangeInput(String),
     Run,
 }
 
-fn init(_url: Url, _orders: &mut impl Orders<AppMsg>) -> AppState {
-    AppState {
+fn init(_url: Url, _orders: &mut impl Orders<Msg>) -> Model {
+    Model {
         source: "".to_string(),
         output: "".to_string(),
     }
 }
 
-fn update(msg: AppMsg, model: &mut AppState, _: &mut impl Orders<AppMsg>) {
+fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg>) {
     match msg {
-        AppMsg::ChangeInput(new_source) => model.source = new_source,
-        AppMsg::Run => {
+        Msg::ChangeInput(new_source) => {
+            model.source = new_source;
+            orders.send_msg(Msg::Run);
```

Comment:
Is there a way to, instead of directly sending the message, we can wait for the user to finish typing before launching the `Msg::Run` ? I mean, with a delay of 200/300ms after the latest keypress ?

I have the feeling that this is going to cause the output to display an error when typing which isn't convenient

Reply:
I've updated, set the delay to 1sec. 
You can change it by modifying `WAIT_TIME` constant.